packages/component-base/src/tooltip-controller.d.ts

```diff
@@ -23,6 +23,12 @@ type TooltipPosition =
  * A controller that manages the slotted tooltip element.
  */
 export class TooltipController extends SlotController {
+  /**
+   * An HTML element to attach the tooltip to.
+   * When not set, defaults to `target`.
+   */
+  ariaTarget: HTMLElement;
```

Comment:
Could there be some other aria attribute that we might need to delegate to some other element at some point? I think the current name is good, just checking if we can identify a reason why it should be more specific.

Reply:
Current name is chosen to be aligned with `ariaTarget` that is used by `FieldMixin`:

https://github.com/vaadin/web-components/blob/fd22eae0fb647997f0e1af4bb25a9bab87d6b505/packages/field-base/src/field-mixin.js#L25-L29

I don't see a need to pick a different name, especially in case of tooltip.